src/main/java/org/example/datastructures/linkedlist/CircularLinkedList/circularLinkedList.java

```diff
@@ -0,0 +1,117 @@
+package LinkedList.CircularLinkedList;
```

Comment:
Please align naming convention for package and class according to this : https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html